src/pages/studyView/addChartButton/customCaseSelection/CustomCaseSelection.tsx

```diff
@@ -68,7 +77,24 @@ export default class CustomCaseSelection extends React.Component<ICustomCaseSele
     get newChartInfo(): NewChart {
         return {
             name: this.chartName ? this.chartName : this.props.getDefaultChartName ? this.props.getDefaultChartName() : '',
-            groups: this.result.validationResult.error.length === 0 ? this.result.groups : []
+            groups: this.result.validationResult.error.length === 0 ? this.result.groups : [],
+            chartType: this.chartType
+        }
+    }
+
+    @computed
+    get chartTypeDropdownTitle(): string {
+        return `Chart Type (${this.chartTypeDisplayName})`;
+    }
+
+    @computed
+    get chartTypeDisplayName(): string {
+        //we just have two types for custom data, it should be pie chart or bar chart
+        if (this.chartType === ChartTypeEnum.PIE_CHART) {
+            return 'Pie chart';
```

Comment:
We don't have much of the use case yet. But I think the display name should be part of a chart meta info.
For now you can use `ChartTypeNameEnum` at least.